### PR TITLE
Add public thanks and recap post for Portland 2019

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: docs/conf/portland/2019/news/thanks-recap.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+.. post:: May 30, 2019
+   :tags: recap, thanks, portland-2019
+
+Conference highlights and thanks to the community!
+--------------------------------------------------
+
+Hey everyone,
+
+The Write the Docs team is now emerging from the post-conference
+recovery period, and we wanted to send out a quick email to thank
```
Comment (mxsasha) [inline docs/conf/portland/2019/news/thanks-recap.rst]: The blog post is not an e-mail. Maybe just "and we wanted to thank everyone for helping" ?

**Review by mxsasha — APPROVED** [review]
Looks good! Left one note.

File: docs/conf/portland/2019/news/thanks-recap.rst (hunk 1 above)
Comment (plaindocs) [reply docs/conf/portland/2019/news/thanks-recap.rst]: Or just send a quick thank you

File: docs/conf/portland/2019/news/thanks-recap.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+At over 500 people, this year's conference was our biggest yet! We had
+the usual mix of main stage and lightning talks, unconference sessions,
+writing sprints, and social events. We were also pleased with the
+success of a recent additions, including livestreaming the
+talks, running real-time captioning for the main stage, and hosting an
+in-person job fair. 
+
+Overall, we were so excited to see another conference come together and
+see our awesome community gather out in the physical world.
```
Comment (plaindocs) [inline docs/conf/portland/2019/news/thanks-recap.rst]: See our awesome community come together, maybe?

File: docs/conf/portland/2019/news/thanks-recap.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+That being said, we're also very conscious of the fact that just because nothing was reported doesn't mean nothing happened. If there was any personal conduct or other aspect of the conference experience that did not meet the expectations of behavior set out in the CoC, please email us at `conduct@writethedocs.org <mailto:conduct@writethedocs.org>`_.
+
+Ready for more?
+===============
+
+Don’t forget our `Vilnius conference <http://www.writethedocs.org/conf/vilnius/2019/>`_ next week,
+`Prague conference <https://www.writethedocs.org/conf/prague/2019/>`__ again in
+September, and our `Australia conference <https://www.writethedocs.org/conf/australia/2019/>`__ in
+November. Tickets are on sale for Prague. The CFP and tickets for the
+Australia conference will open soon.
```
Comment (plaindocs) [inline docs/conf/portland/2019/news/thanks-recap.rst:74]: Should we start saying which will be live streamed?

**Review by plaindocs — APPROVED** [review]

File: docs/conf/portland/2019/news/thanks-recap.rst (hunk 3 above)
Comment (ericholscher) [reply docs/conf/portland/2019/news/thanks-recap.rst:74]: I think we're fine for now. Maybe in the future.

**Pull request merged**